CONTRIBUTING.md

```diff
@@ -66,11 +66,11 @@ The `issue-number` is the number associated with the ticket / issue on the [issu
 
 ### Pull Requests
 
-This is refering to the pull request title which should follow the same format as the branch name for simplicity. So a branch with the name of `doc/10_add_readme_info` should have a pull request with the title of `doc/10_add_readme_info`
+This is refering to the pull request title which should describe in short what the PR is doing as well as including the issue id. An example is `#22 new api endpoint for likes`
 
```

Comment:
Should probably state to include `close / fixes [issue number]` to both close and link the associated issue.